ElectronClient/gui/Header/Header.jsx

```diff
@@ -247,13 +256,35 @@ class HeaderComponent extends React.Component {
 			</a>
 		);
 
+
+		const fuzzyToggle = (
+			<div key={key + state.fuzzy.toString()}>
```

Comment:
Could you clarify why the key is set like this? Is it to force an update of the component? If that's needed it means there's some issue with the way state is handled. In fact I've just noticed that you don't even need to set a key here normally.

Reply:
Could you fix your code so that this hack is not needed please? (in general, new code shouldn't be based on old hacks)